straxen/plugins/nveto_recorder.py

```diff
@@ -0,0 +1,374 @@
+import numba
+import numpy as np
+from scipy.ndimage import convolve1d
+from frozendict import frozendict
+
+import strax
+import straxen
+
+export, __all__ = strax.exporter()
+
+__all__ = ['nVETORecorder']
+
+
+@strax.takes_config(
+    strax.Option('coincidence_level', type=int, default=4,
```

Comment:
Strax uses a single configuration dictionary for all registered plugins, so generic option names like this are a bit dangerous. Maybe e.g. `nv_minimum_recording_coincidence` ? If you use the option often in code you can assign it to a local variable or attribute to avoid repeating the long name. 

Maybe you could go through the other options in your plugins with this in mind? I can see a few more that might give troubles.

Reply:
Done